cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,195 @@
+
+        {# ADD THE FILTER CONTROLS. #}
+
+        {% set fragment_id = 'o-filterable-list-controls' %}
+        {{ filterable_list_controls.render(value, form, fragment_id) }}
+
+        {# ADD THE NOTIFICATION, IF NEEDED. #}
+
+        {% set notification_opts = {
+            'type': 'default',
+            'is_visible': false,
+            'message': '',
+            'explanation': ''
+        } %}
+
+        {% for field in form %}
+            {% if field.errors %}
+                {% for error in field.errors %}
+                    {# The if statement here allows access to the
+                       notification_opts variable outside of the for loop.
+                    #}
+                    {% if notification_opts.update({
+                        'type': 'error',
+                        'is_visible': true,
+                        'message': notification_opts.message + field.label + ': ' + error + '<br>' | safe
+                    }) %}{%endif%}
+                {% endfor %}
+            {% endif %}
+        {% endfor %}
+        {% if 'filterable-list' in value.form_type and posts is defined %}
+            {% if form.is_valid() %}
+                {% set count = posts.paginator.count %}
+                {% set notification_is_visible = true %}
+                {% if count == 0 %}
+                    {# The if statement here allows access to the
+                        notification_opts variable outside of the for loop.
+                    #}
+                    {% if notification_opts.update({
+                        'type': 'warning',
+                        'is_visible': true,
+                        'message': 'Sorry, there were no results based on your filter selections.',
+                        'explanation': 'Please reset the filter or change your selections and try again.'
+                    }) %}{%endif%}
+                {% else %}
+                    {# The if statement here allows access to the
+                        notification_opts variable outside of the for loop.
+                    #}
+                    {% if notification_opts.update({
+                        'type': 'success',
+                        'is_visible': true,
+                        'message': count ~ ' filtered result' ~ count|pluralize('s')
+                    }) %}{%endif%}
+                {% endif %}
+            {% endif %}
+        {% endif %}
+
+        <div class="o-filterable-list_notification
+                    block
+                    block__padded
+                    block__sub
+                    {% if not has_active_filters %}
+                    u-hidden
+                    {% endif %}">
+            {{ notification.render(
+                notification_opts.type,
+                notification_opts.is_visible,
+                notification_opts.message,
+                notification_opts.explanation
+            ) }}
+        </div>
+
+        {# ADD THE FILTER RESULTS, IF THERE ARE ANY. #}
```

Comment:
Can we add an `if` statement here so the empty `section` is not rendered if there are no results?

Reply:
Added in https://github.com/cfpb/cfgov-refresh/pull/4771/commits/b322bf00c90571c5ca021608bd29adc5308fe4e2